db/seeds.rb

```diff
@@ -94,26 +94,36 @@
     email: 'testytesterson@gmail.com',
     item_type: Item.gift_card,
     refunded: false,
+    single_use: false,
```

Comment:
I think the default value for this is false, so I can take it out if we want

Reply:
I think it's nice to be explicit for everyone reading the file. What do you think @justintmckibben ?